### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by hnryjmes**
Frontend changes to add pagination, including:
- basic logic to hide/show buttons in the template
- CSS styling to match Data Hub pagination

Screenshot:

Screenshot 2020-03-16 at 15 29 11

### CODE REVIEW

File: app/enquiries/templates/enquiry_list.html [hunk 1]
```diff
@@ -73,7 +73,39 @@ <h3>
...
+                <ol class="pagination-list">
+                    {%if previous is not None %}
+                    <li class="pagination-list-item">
+                        <a class="pagination-link" href="{{ previous }}">Previous</a>
+                    </li>
+                    {%endif%}
+
+                    {% for page in page_range %}
+
+                    {% if page|stringformat:"s" == current_page %}
```
Comment (currycoder) [inline app/enquiries/templates/enquiry_list.html]: I think it would be nice if the template code was indented in for/if blocks.

File: app/enquiries/tests/test_views.py [hunk 2]
```diff
@@ -70,7 +70,7 @@ def canned_enquiry():
 
 REST_FRAMEWORK_TEST = {
     "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.PageNumberPagination",
-    "PAGE_SIZE": 2,
+    "PAGE_SIZE": 10,
```
Comment (currycoder) [inline app/enquiries/tests/test_views.py]: Why have these values been adjusted?  I believe that the original idea behind keeping these low was to easily test the edge cases around pagination without having to create a lot of data per test.  e.g. create enough entities for a first full page of items and a second page with just one record on it.

Comment (hnryjmes) [reply app/enquiries/tests/test_views.py]: @saruniitr see above, the reason for this was a bug we saw in development, can you give more information?

Comment (saruniitr) [reply app/enquiries/tests/test_views.py]: Yes this was a temporary change to unblock frontend. The fixes are done in backend and they were under review.
Now they are merged and this doesn't exist in `master`. Updating with `master` will fix this.

Comment (saruniitr) [reply app/enquiries/tests/test_views.py]: Reverted temporary fix and updated with `master`.

**Comment by saruniitr** [comment]
Thanks for reviews, two approvals received, merging it.
